js/main.js

```diff
@@ -0,0 +1,65 @@
+
+// so that we can use anywhere, will be used for pretty much everything
+let storage = window.localStorage;
+
+let createTodo = () => {
+  // prevent reload
+  event.preventDefault(); // VScode says that event is deprecated but nothing else works...
```

Comment:
This happens because you use the global event property. You should change the function to `let createTodo = event => {...}`. Then you can use event.preventDefault